platform_umbrella/apps/common_core/lib/common_core/resources/karpenter.ex

```diff
@@ -399,8 +401,9 @@ defmodule CommonCore.Resources.Karpenter do
     cluster_name = Core.config_field(state, :cluster_name)
 
     spec = %{
-      "amiFamily" => "AL2",
       "role" => battery.config.node_role_name,
+      "amiFamily" => "AL2",
+      "amiSelectorTerms" => [%{"alias" => "al2@v20250123"}],
```

Comment:
This will need to change / we'll need to figure out a way to update the version that should be used. We could use `@latest` but that has its own set of issues.

Reply:
Should this be a defaultable field on the config, then?

Reply:
The concern with that is that it would need to be changed to the correct version by the user as they upgrade, I think.

Reply:
If they don't set the field, they will get the default of the running version. If they do set the `ami_selector` (or whatever) field then they are responsible for making sure it works. That seems reasonable since that ami seems to be tied to the running karpenter version